examples/platform_channel/macos/Runner/MainFlutterWindow.swift

```diff
@@ -5,37 +5,54 @@
 import Cocoa
 import FlutterMacOS
 
-class MainFlutterWindow: NSWindow, FlutterStreamHandler, PowerSourceStateChangeDelegate {
+enum ChannelName {
+  static let battery = "samples.flutter.io/battery"
+  static let charging = "samples.flutter.io/charging"
+}
+
+enum BatteryState {
+  static let charging = "charging"
+  static let discharging = "discharging"
+  static let unavailable = "UNAVAILABLE"
+}
+
+enum ErrorCode {
+  static let noBattery = "NO_BATTERY"
+  static let unavailable = "UNAVAILABLE"
+}
```

Comment:
these can be string backed enums. For example:

```
enum BatteryState: String {
  case charging
  case discharging
  case unavailable
}

enum ErrorCode: String {
  case noBarry = "NO_BATTERY"
  case unavailable = "UNAVAILABLE" 
}
```

Reply:
Added to make it similar to the iOS swift example, but it's inconsistent with the rest of the samples, irrelevant to demonstrating channels, and makes it so the useful code is all hidden below the fold, and you've got to scroll down to get to the actual bit that we want to demonstrate. Deleted instead for similarities to all the other samples, bar iOS Swift.